lib/Translation/ImporterService/Importer/DataObjectImporter.php

```diff
@@ -19,6 +19,8 @@
 use Pimcore\Translation\AttributeSet\Attribute;
 use Pimcore\Translation\ExportDataExtractorService\DataExtractor\DataObjectDataExtractor;
 
+use function explode;
```

Comment:
can you revert this line please?

Reply:
I can do but it's considered best practice in symfony.